front/README.md

```diff
@@ -39,36 +39,16 @@ Launches end to end tests.
 
 It requires:
 
-- Backend containers to be up: `docker compose build core redis editoast postgres`
-- Running front
-- [`poetry`](https://python-poetry.org/docs/#installation) to be installed
-
-Locally, to use the same configuration as the CI, you must change the environment variable to
-`CI=true`.
-
-This command will execute all tests defined. The command can be changed in
-[`test_e2e.py`](../tests/tests/test_e2e.py) to execute only one test with or without debug mode.
-`test_e2e.py` is executing [playwright command](https://playwright.dev/docs/test-cli) in a python
-subprocess. Example of modification to apply to launch only `stdcm-page.spec.ts` in debug mode with
-chromium:
-
-```diff
-    result = subprocess.run(
-        [
-            "yarn",
-            "--cwd",
-            "front",
-            "playwright",
-            "test"
-+           "stdcm-page.spec.ts",
-            "--reporter=line",
-+           "--debug",
-+           "--project=chromium"
-        ],
-        cwd=Path(__file__).parents[2],
-        check=False,
-    )
-```
+- Install playwright dependencies `npx --prefix ./front/ playwright install --with-deps`
+- Backend containers to be up: `TAG=dev docker compose up --no-build --detach redis postgres gateway core editoast`
+- Running front with `TAG=dev docker compose up --build --detach front`
+
+Now you can run the test with `cd front/ && yarn e2e-tests`.
+
+:warning: You may be tempted to launch the frontend locally with `yarn start` but this won't work
```

Comment:
If not locally, where is the frontend launched? The word may be ambiguous

Reply:
and you can also launch the project on Linux distrib with the osrd-compose.sh script, which allows you to use yarn start, maybe we should add a comment about that

Reply:
See https://github.com/osrd-project/osrd/compare/418a2f76ab65b8a02785c2fcb9d84c2bc18a2402..674abc14156b3e0d455c7a5c7da06fcc54b8b928 for the last version, I've also used the nice "Caution blocks" from Github Markdown (just discovered them).

![image](https://github.com/osrd-project/osrd/assets/2520723/abeccbe3-c6ac-4583-bdbb-5cb20c90ca9c)